src/hooks/use-locale-number.test.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { ObjectListProvider } from '../providers'
+import useLocaleNumber from './use-locale-number'
+import type { NumberToLocaleParameters } from '../types'
+
+type Formatter = {
+  parameters?: NumberToLocaleParameters
+  value: number
+}
+
+const CurrencyFormatter: React.FC<Formatter> = ({ parameters, value }: Formatter) => {
+  const { format } = useLocaleNumber({ ...parameters, currency: true })
+  return <div>{format(value)}</div>
+}
+
+const NumberFormatter: React.FC<Formatter> = ({ parameters, value }: Formatter) => {
+  const { format } = useLocaleNumber({ ...parameters })
+  return <div>{format(value)}</div>
+}
+
+describe('Convert Number To Locale', () => {
```

Comment:
Question on consistency - we use lowercase in our other test files, but Title Case throughout this one. I think the convention is the former. Could we update to lowercase to stay with convention?

```suggestion
describe('convert number to locale', () => {
```